styles/main.css

```diff
@@ -0,0 +1,415 @@
+  height: 0.1rem;
+  background-color: var(--gray-one);
+}
+
+.man-one {
+  position: static;
+}
+
+.min-images {
+  margin-top: 0.3rem;
+  width: 4rem;
+  height: 4rem;
+}
+
+.opinion a + a {
+  border-top: 1px solid var(--gray-one);
+}
+
+.separator {
+  border-bottom: var(--big-border);
+}
+
+.main-debate-container::after,
+.main-debate-container::before {
+  margin: 0.5rem;
+  content: "";
+  display: block;
+  height: 0.3rem;
+  background-color: var(--gray-one);
+}
+
+.min-imge-more {
+  margin-right: 1rem;
+  width: 6rem;
+  height: 6rem;
+}
+
+.resize-heading-h3 {
+  font-size: 0.9rem;
+}
+
+.small-p {
+  font-size: 0.8rem;
+}
+
+.first-section-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-areas: "top-story" "featured-stories" "debate-container" "more-stories" "my-t-culture";
+}
+
+.top-story {
+  grid-area: top-story;
+}
+
+.featured-stories {
+  grid-area: featured-stories;
+}
+
+.my-t-culture {
+  grid-area: my-t-culture;
+}
+
+.more-stories {
+  grid-area: more-stories;
+}
+
+.border-top {
+  padding-top: 1rem;
+}
+
+.news-signup {
+  margin: auto;
+  width: 15rem;
+  height: 10rem;
+}
+
+.n-img {
+  width: 2rem;
+  height: 2rem;
+  margin: 0.5rem;
+}
+
+.news-signup h2 {
+  font-size: 1.05rem;
+  font-weight: bolder;
+  line-height: 1rem;
+  margin: 0.5rem;
+  height: 3rem;
+}
+
+.news-signup button {
+  width: 13rem;
+  margin: auto;
+  margin-top: 0.5rem;
+}
+
+.magazine-section {
+  width: auto;
+}
+
+.magazine-section h2 {
+  font-size: 1.3rem;
+  font-weight: bolder;
+}
+
+.magazine-cover img {
+  width: 100%;
+}
+
+.last-section h6 {
+  line-height: 1rem;
+  font-size: small;
+}
+
+.promotion-section img {
+  max-width: 10rem;
+}
+
+.promotion-section {
+  line-height: 1.1rem;
+}
+
+.promotion-section button {
+  margin: auto;
+}
+
+.mini-cover-one {
+  top: 0.8rem;
+  left: 4rem;
+}
+
+.women-conianer {
+  margin: auto;
+  padding-right: 15rem;
+  width: 10rem;
+}
+
+.buttom-links li + li {
+  padding: 0 0.3rem;
+}
+
+.bg-phone {
+  background-image: url('../assets/images/phone.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  max-width: 100vw;
+  height: 10rem;
+  margin: auto;
+}
+
+.footer-header i {
+  color: gray;
+  font-size: x-large;
+  padding: 0.3rem;
+}
+
+.footer-header img {
+  width: 15rem;
+}
+
+.footer-header span {
+  font-size: 0.8rem;
+}
+
+@media screen and (min-width: 768px) {
+  .first-section-grid {
+    grid-template-columns: 3fr 4fr;
+    grid-template-rows: 33rem 1fr 141rem;
+    grid-template-areas:
+      "featured-stories top-story"
+      "featured-stories my-t-culture"
+      "featured-stories more-stories"
+      "debate-container debate-container";
+  }
+
+  .my-t-culture img {
+    max-width: 9rem;
+    height: 8rem;
+    padding: 0.1rem 0.5rem;
+  }
+
+  .my-t-culture article {
+    max-width: 8rem;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 1rem;
```

Comment:
Apply bootstrap classes wherever you can, please add bootstrap classes and reduce your CSS file.